### ISSUE TITLE
Adding a second external node to a project loops forever

### ISSUE THREAD

**Issue by obourdon** [squarescale/squarescale-cli#167]
Using the [new edge demo script ](https://github.com/squarescale/sqsc-demo-scripts/blob/master/sqsc_edge_demo.sh), the second external node addition keeps waiting as follows:

```
Waiting for project to be able to schedule containers
Project sqsc-edge-demo READY to schedule containers
Creating external node edge1 with IP 1.1.1.1 in sqsc-edge-demo...
... done
Successfully added external node 'edge1' to project '8a83351d-872d-4c6d-a1fc-f80b0aba98c2'
◓ wait for external node add
... done
edge1
Waiting for project to be able to schedule containers
Project sqsc-edge-demo is not ready to schedule any containers yet
Project sqsc-edge-demo is not ready to schedule any containers yet
...
Project sqsc-edge-demo READY to schedule containers
Creating external node edge2 with IP 2.2.2.2 in sqsc-edge-demo...
... done
Successfully added external node 'edge2' to project '8a83351d-872d-4c6d-a1fc-f80b0aba98c2'
◒ wait for external node add
```

**Comment by obourdon** [comment]
Seems like [the following code](https://github.com/squarescale/squarescale-cli/blob/master/squarescale/external_nodes.go#L94) is wrong for external nodes (most probably copied over from extra nodes [former stateful nodes])

In fact, the 1st external node addition addition returns `provisionned` as the status of the new external node whereas the 2nd addition returns `inconsistent`

However, as per intrinsic nature, external nodes can be on or off-line therefore this should not be checked at all

**Comment by obourdon** [comment]
Please note that this can be avoided by adding the `-nowait` flag to `sqsc external-node add ...`

**Comment by obourdon** [comment]
Tentative fix available in branch `edge-demo-fixes`

**Comment by obourdon** [comment]
Released in 1.1.6
